packages/browser/src/plugins/ajs-destination/index.ts

```diff
@@ -226,7 +226,6 @@ export class LegacyDestination implements DestinationPlugin {
             type: 'Dropped by plan',
           })
         )
-        return ctx
```

Comment:
This is ok to remove because our event queue doesn't do anything with the returned ctx?

Reply:
My editor actually autoremoved these because ctx.cancel throws an error, so this is dead code:

<img width="600" alt="image" src="https://github.com/segmentio/analytics-next/assets/5115498/a7e80c63-27ca-4995-88d5-4e82a97649ea">